### PR TITLE
dynamically reload aggregator cert

### PR DESCRIPTION
**Pull request by deads2k**
Adds one commit to dynamically reload aggregator certs

/assign @sttts @mfojtik

### CODE REVIEW

File: vendor/k8s.io/kubernetes/staging/src/k8s.io/kube-aggregator/pkg/apiserver/apiserver.go [hunk 1]
```diff
@@ -21,6 +21,8 @@ import (
 	"strings"
 	"time"
 
+	"k8s.io/apiserver/pkg/server/certs"
```
Comment (sttts) [inline vendor/k8s.io/kubernetes/staging/src/k8s.io/kube-aggregator/pkg/apiserver/apiserver.go]: nit: order

File: vendor/k8s.io/kubernetes/staging/src/k8s.io/kube-aggregator/pkg/apiserver/apiservice_controller.go [hunk 2]
```diff
@@ -165,3 +167,15 @@ func (c *APIServiceRegistrationController) deleteAPIService(obj interface{}) {
 	glog.V(4).Infof("Deleting %q", castObj.Name)
 	c.enqueue(castObj)
 }
+
+// resyncAll queues all apiservices to be rehandled.
+func (c *APIServiceRegistrationController) resyncAll() {
```
Comment (sttts) [inline vendor/k8s.io/kubernetes/staging/src/k8s.io/kube-aggregator/pkg/apiserver/apiservice_controller.go]: don't see where it is used.

Comment (deads2k) [reply vendor/k8s.io/kubernetes/staging/src/k8s.io/kube-aggregator/pkg/apiserver/apiservice_controller.go]: > don't see where it is used.

awkward....

File: vendor/k8s.io/kubernetes/staging/src/k8s.io/kube-aggregator/pkg/apiserver/apiserver.go [hunk 3]
```diff
@@ -187,17 +185,28 @@ func (c completedConfig) NewWithDelegate(delegationTarget genericapiserver.Deleg
 	s.GenericAPIServer.Handler.NonGoRestfulMux.UnlistedHandle("/apis/", apisHandler)
 
 	apiserviceRegistrationController := NewAPIServiceRegistrationController(informerFactory.Apiregistration().InternalVersion().APIServices(), s)
+	aggregatorProxyCerts := certs.NewDynamicCertKeyPairLoader(c.ExtraConfig.ProxyClientCert, c.ExtraConfig.ProxyClientKey, apiserviceRegistrationController.resyncAll)
```
Comment (deads2k) [inline vendor/k8s.io/kubernetes/staging/src/k8s.io/kube-aggregator/pkg/apiserver/apiserver.go]: @sttts wired usage here

**Comment by deads2k** [comment]
Since the test failed, I rebased and added the commit for my IOU on docs.

**Comment by deads2k** [comment]
known DC flake.  I'm tempted, but 

/retest

**Comment by deads2k** [comment]
None of the comments were substantive.  Going for this to release the operator pull and stop restarting

**Comment by deads2k** [comment]
nothing that merged interacts

**Pull request merged**